res/controllers/Denon-MC7000-scripts.js

```diff
@@ -724,36 +691,29 @@ MC7000.getPrevRateRange = function(currRateRange) {
     }
     return MC7000.negRateRanges[0];
 };
-        
+
 /* LEDs for VuMeter */
 // VuMeters only for Channel 1-4 / Master is on Hardware
 MC7000.VuMeter = function(value, group) {
-	var deckNumber = script.deckFromGroup(group),
-	    peak = 0x76, // where the red LED starts (clipping indicator)
-        level = value*value*value*value*0x69;
+    var VuMeterLEDPeakValue = 0x76,
```

Comment:
there is not really a reason for redefining this value inside the function all the time. Especially since its a constant so I'd define it outside this function once.

Reply:
OK. Then I gonna put it at the beginning of the script to the other variables.